pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl

```diff
@@ -0,0 +1,66 @@
+version 1.0
+
+import "WGSFromBam.wdl" as WGSFromBam
+
+
+workflow WGSMultipleSamplesFromBam {
+
+  String pipeline_version = "1.0.0"
+
+  input {
+    File sample_map
+
+    DNASeqSingleSampleReferences references
```

Comment:
Don't you need to import these structs?

Reply:
I guess the equivalent `*SamplesFromFastqs.wdl` doesn't, so this is fine.

Reply:
Yep, it gets imported implicitly with `import "WGSFromBam.wdl" as WGSFromBam`